en/micro-integrator/docs/references/connectors/salesforce-rest-connector/sf-rest-connector-example.md

```diff
@@ -21,16 +26,123 @@ Follow these steps to set up the ESB Solution Project and the Connector Exporter
+    
+    Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then drag and drop the `init` operation into the Design pane.
+        
+    <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-init.png" title="Drag and drop init operation" width="800" alt="Drag and drop init operation"/>
+        
+    Add the property values into the `init` operation as shown bellow. Replace the `clientSecret`, `clientId`, `accessToken`, `refreshToken` with obtained values from above steps.
+      
+    - **clientSecret** : Value of your client secret given when you registered your application with Salesforce.
+    - **clientId** : Value of your client ID given when you registered your application with Salesforce.
+    - **accessToken** : Value of the access token to access the API via request.
+    - **refreshToken** : Value of the refresh token.
+       
+    <img src="../../../../assets/img/connectors/salesforce-api-init-operation-sequnce.png" title="Add values to the init operation" width="800" alt="Add values to the init operation"/>
+    
+    The above four parameters are saved to the properties section and add the values into the `init` operation.
+    
+    <img src="../../../../assets/img/connectors/salesforce-api-create-operation-sequnce.png" title="Add values to the create operation" width="800" alt="Add values to the create operation"/>
+     
+    2.2 Setting up the created operation
+
+    Then we will setup the `create` sequence configurations. In this operation we are going to create a `sObjects` in the Salesforce account. An `SObject` represents a specific table in the database that you can discretely query. It describes the individual metadata for the specified object. Please find the `create` operation parameters listed here.
+       
+    - **sObjectName** : Name of the sObject which we need to create in the salesforce.
+    - **fieldAndValue** : The field and value need to store in the created salesforce sObject.
+        
+    While invoking the api the above two parameters values are coming as a user input.
+    
+    Navigate into the **Palette** pane and select the graphical operations icons listed under **Salesforce Connector** section. Then drag and drop the `create` operation into the Design pane.
+    
+     <img src="../../../../assets/img/connectors/salesforce-drag-and-drop-create.png" title="Drag and drop create operation" width="800" alt="Drag and drop create operations"/>
+    
+    To get the input values in to the API we can use the [property mediator](https://ei.docs.wso2.com/en/next/micro-integrator/references/mediators/property-Mediator/) .
+
+    Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown bellow.
```

Comment:
To get the input values in to the API we can use the [property mediator](https://ei.docs.wso2.com/en/next/micro-integrator/references/mediators/property-Mediator/) .

    Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown bellow.

These two points can be merged.

Reply:
Merged.